.github/README.md

```diff
@@ -0,0 +1,142 @@
+# FirebaseManager Package
+
+This repository contains all utility functions that simplify transactions with Firebase products like Cloud Firestore, Cloud Storage or Authentication, wrapped in a Swift Package.
+It is made by **[SPACE SQUAD](https://www.spacesquad.de)**! We make great software with ♥️ in Berlin.
```

Comment:
I would prefer a separate line or paragraph for the SSQ mention, especially the logo should not be inline.

Reply:
Fixed